hasher-matcher-actioner/terraform/migrations/README.md

```diff
@@ -1,8 +1,8 @@
 # What are migrations
 
-Migrations are commands to run when installing a new version of HMA. This includes both a fresh install or an upgrade of an older version. 
+Migrations are commands to run when installing a new version of HMA. 
```

Comment:
blocking q: Wait, is the note about being a fresh install vs upgrade not the case anymore?

Reply:
This README used to say it was for both, right now the README is about an empty module but if used it would be for upgrading older version.